lib/plugins/jexlParser.js

```diff
@@ -185,8 +150,53 @@ function processExpressionAttributes(typeInformation, list, context) {
         .map(expressionApplier(context, typeInformation));
 }
 
+function checkTransformationMap(tranformsMap) {
+    //[error,message, resultMap] = expressionParser.checkTransformationMap(niceMap);
+    let error = null;
+    let message = 'No trasformations were added to JEXL Parser';
+    let resultMap = {};
+
+    if (typeof tranformsMap != 'object') {
+        error = true;
+    } else if (
+        tranformsMap === null ||
+        (tranformsMap && Object.keys(tranformsMap).length === 0 && tranformsMap.constructor === Object)
+    ) {
+        //default
+    } else {
+        //detecting wrong transformations
+        let wrongList = [];
+        for (var transformation in tranformsMap) {
+            if (typeof tranformsMap[transformation] != 'function') {
+                wrongList.push(transformation);
+            } else {
+                resultMap[transformation] = tranformsMap[transformation];
+            }
+        }
+        if (wrongList.length === 0) {
+            message = 'Trasformations can be added to JEXL parser';
+        } else {
+            message = wrongList.toString() + ' must be a function';
+        }
+    }
+    return [error, message, resultMap];
+}
+
+function setTransforms(configMap) {
+    //Check provided transforms since they are provided by user
```

Comment:
Maybe is a good idea to document what happen when name conflict occurs, something like this:

```suggestion
    //Check provided transforms since they are provided by user
    //Note that in case of name conflict, baseTransformsMap would take precedence
```